bitrepository-reference-settings/src/main/resources/xsd/ReferenceSettings.xsd

```diff
@@ -398,6 +398,14 @@
                 </xs:annotation>
               </xs:element>
               <xs:element ref="PillarType"/>
+              <xs:element name="PillarDeleteFileApprover" type="xs:string">
+                <xs:annotation>
+                  <xs:documentation>
+                    Name/email/contact-info of the person responsible for approving any deletions on this pillar prior
```

Comment:
Might we want to relax it a bit? “Free text; typically name/email address/contact info of ...”.